packages/ui-components/src/Input/EmailPasswordInput.tsx

```diff
@@ -1,81 +1,137 @@
 import React, { useState, useEffect } from "react";
-import { BaseTextInput, AllProps } from "./TextTypeInput";
+import { BaseTextInput, TextInputProps } from "./TextTypeInput";
 import "./EmailPassword.module.scss";
 import { Icon } from "../Icon/Icon";
+import { FieldRenderProps } from "react-final-form";
 
-export type ExistingPasswordProps = Omit<AllProps, "multiline">;
+export interface Validator {
+  fn: (val: string) => boolean;
+  label: string;
+}
 
-// since we use at most one validator currently
-// we only want to accept one validator label
-export type NewPasswordProps = Omit<
-  ExistingPasswordProps & { validatorLabel: string },
-  "forgotPasswordLinkElement"
->;
-export type EmailProps = NewPasswordProps;
+export interface PasswordProps
+  extends Partial<FieldRenderProps<"text", HTMLInputElement>>,
+    TextInputProps {
```

Comment:
moved the `FieldRenderProps` (which define `input` and `meta`) here, since the other Input components don't need them